### PR TITLE
fix multiple bugs in scrot's -C option

### PR DESCRIPTION
**Pull request by guijan**
I've carefully read the Xlib manuals and some more, but this PR should be reviewed by someone more familiar with X programming.

### CODE REVIEW

File: src/options.c [hunk 1]
```diff
@@ -474,7 +465,7 @@ void optionsParse(int argc, char *argv[])
             optionsParseStack(optarg);
             break;
         case 'C':
-            optionsParseWindowClassName(optarg);
+            opt.windowClassName = optarg;
```
Comment (N-R-K) [inline src/options.c:468]: Could you make `opt.windowClassName` const in options.h ? Assuming there won't be any issues.

**Comment by N-R-K** [comment]
Unfortunately I cannot test this since the `-k` options seems to require a compositor (and I don't want to go though the hassle of setting one up).

Can you confirm that you did some before/after testing (just in case) @guijan ? If yes, then I'll go ahead and merge this since the code looks OK to me.

**Comment by guijan** [comment]
Yeah, I've tested it. Here's a screenshot I just took.
2023-03-25-073826_956x2144_scrot

**Comment by N-R-K** [comment]
Thanks.

**Pull request merged**